README.md

```diff
@@ -6,7 +6,7 @@ It was produced by the Bibliotheca Anonoma as a replacement for FoolFuuka, to be
 
 ## Installation
 
-Ensure that Python and Pip are installed. Python 2.7 or Python 3.3 and higher is recommended.
+Ensure that Python and Pip are installed. Python 3.8 or higher is recommended.
```

Comment:
I'm a bit reserved on requiring Python 3.8 or higher considering most enterprise level architectures use older versions of Python (usually Python 3.6) and PyPy doesn't support Python 3.6 or higher syntax. 

Ayase was working fine on PyPy 3.6 though, I'll test it again later but maybe we can just change this line to 3.6 if it works the way it should.

Reply:
Works fine. I'll just merge and update documentation.

Reply:
Since the minimum is 3.6.0 according to FastAPI, I tried starting from there but quickly ran into [issue](https://github.com/python/typing/issues/530).

Then I tried **3.6.1** and it worked. So that should be the minimum version on the README.

I also tried PyPy3.6-7.3.1 but it seems like it can't compile `asyncpg` at the moment. [Related issue](https://github.com/MagicStack/asyncpg/issues/369).

If you want PyPy support you could always switch back to sync and drop async. It should be trivial to do.

Reply:
Since the minimum is 3.6.0 according to FastAPI, I tried starting from there but quickly ran into this: `https://github.com/python/typing/issues/530`.

Then I tried **3.6.1** and it worked. So that should be the minimum version on the README.

I also tried PyPy3.6-7.3.1 but it seems like it can't compile `asyncpg` at the moment. Related issue: `https://github.com/MagicStack/asyncpg/issues/369`.

If you want PyPy support you could always switch back to sync and drop async. It should be trivial to do.

Reply:
Since the minimum is 3.6.0 according to FastAPI, I tried starting from there but quickly ran into this: `https://github.com/python/typing/issues/530`.

Then I tried **3.6.1** and it worked. So that should be the minimum version on the README.

I also tried PyPy3.6-7.3.1 but it seems like it can't compile `asyncpg` at the moment. Related issue: `https://github.com/MagicStack/asyncpg/issues/369`.

If you want PyPy support you could always switch back to sync and drop async. It should be trivial to do.

*Side note*  
Man GitHub is so annoying. I don't want my mentions all over other peoples issue trackers.

Reply:
Welcome to the Github Social Network.

Actually I have it running on PyPy right now. I didn't document it, but it works if you have asyncpg install it with CPython and then switch to PyPy.